code/modules/mob/living/carbon/human/species/station/prometheans_vr.dm

```diff
@@ -9,13 +9,7 @@
 		"Rapala", "Neaera", "Stok", "Farwa", "Sobaka",
 		"Wolpin", "Saru", "Sparra")
 
-	heal_rate = 0.2 //As of writing, original was 0.5 - Slows regen speed (bad)
-	hunger_factor = 0.1 //As of writing, original was 0.2 - Slows hunger rate (good)
-	siemens_coefficient = 1 //As of writing, original was 0.4 (bad)
-	active_regen_mult = 0.66 //As of writing, original was 1 (good)
-
 	color_mult = 1
-	mob_size = MOB_MEDIUM //As of writing, original was MOB_SMALL - Allows normal swapping (good)
```

Comment:
This is not a nerf, but rather is there to make Vorestation features (especially size difference mechanics) work properly.